server/models/connection_persister.go

```diff
@@ -49,11 +50,17 @@ func (cp *ConnectionPersister) GetConnections(search, order string, page, pageSi
 	count := int64(0)
 
 	connectionsFetched := []*connections.Connection{}
-
 	query.Table("connections").Count(&count)
-
+	environmentsFetched := []environments.EnvironmentData{}
```

Comment:
Please use from schemas package, checkout environment_persister.go file for implementation, and here is where environment is defined - https://github.com/meshery/schemas/blob/master/models/v1beta1/environment.go#L12
```suggestion
	environmentsFetched := []v1beta1.Environment{}
```

Reply:
Ping me when this is done

Reply:
> environmentsFetched := []v1beta1.Environment{}

This environment struct is not compatible with the connection.Environment struct which is `[]environments.EnvironmentData`

Reply:
I see a complete migration of connection needed to make sure we are using v1beta1, never mind you can leave this review thanks for confirming